egs2/clotho_v2/asr1/run_pt.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+# Runs pre-training
+set -euo pipefail
+
+timestamp=$(date "+%Y%m%d.%H%M%S")
+
+# wandb_init_args="--use_wandb true --wandb_project DCASE_AAC --wandb_model_log_interval 0"
+wandb_init_args=""
+other_args="$@"
+
+
+./asr.sh \
+    --asr_tag pt.${timestamp} \
+    --asr_speech_fold_length 1600 \
+    --feats_normalize uttmvn \
+    --stage 1 \
+    --stop_stage 13 \
```

Comment:
why was it stopped at stage 13?

Reply:
Willl update this to default values. 14 is model packing and 15 is uploading to HF (I did these separately as my setup wasn't letting me upload via terminal, currently idk why).

Reply:
model uploading does not happen in a default setup
https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/asr2/asr2.sh#L34
So, you do not need it anyway.